stepflow-rs/crates/stepflow-core/src/workflow/flow.rs

```diff
@@ -257,10 +257,89 @@ impl JsonSchema for FlowRef {
         Flow::inline_schema()
     }
 }
+/// Configuration for testing a workflow.
```

Comment:
Move this code in flow around so we have a single test block. I thought this was why the test wasn't running. I don't think it was, but still better to have a single test block.